src/ixdat/readers/ec_ms.py

```diff
@@ -1,9 +1,73 @@
 from . import TECHNIQUE_CLASSES
+import pickle
```

Comment:
the module name clashes with the technique module name. They're in different places in ixdat but it could get confusing at some point. Maybe this one could be called `ec_ms_pkl`.